CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+* Package: openSUSE-release-tools-pkglistgen
+* Usage: gocd
+
+#### container_cleaner
+
+Cleans up the `maintenance_release` projects for containers
+
+* Sources: [container_cleaner.py](container_cleaner.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### metrics
+
+Generates insightful metrics from relevant OBS and annotation data, based on InfluxDB and Grafana.
+
+* Sources: [metrics.py](metrics.py)
+* Documentation: [docs/metrics.md](./docs/metrics.md)
+* Package: openSUSE-release-tools-metrics
+* Usage: ?
+
+#### metrics-access
+
+Ingests `download.opensuse.org` Apache access logs and generate metrics. It is composed of a PHP.
+script and a set of [systemd units](systemd).
+
+* Sources: [metrics/access/aggregate.php](metrics/access/aggregate.php)
+* Documentation: [docs/metrics.md](./docs/metrics.md)
+* Package: openSUSE-release-tools-metrics-access
+* Usage: ?
+
+#### totest-manager
+
+Releases distribution snapshots to openQA and publishes if the result is positive.
+
+* Sources: [totest-manager.py](totest-manager.py) and [ttm](ttm)
+* Documentation: [ttm/README.md](ttm/README.md)
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### unmaintained
+
+Finds unmaintained binaries sourced from SLE.
+
+* Sources: [unmaintained.py](unmaintained.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### sync-rebuild
+
+Syncs openSUSE:Factory and openSUSE:Factory:Rebuild.
+
+* Sources: [sync-rebuild.py](sync-rebuild.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### bugowner
+
+Manages bugowner information
+
+* Sources: [bugowner.py](bugowner.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### bs_mirrorfull
+
+Mirrors repositories from build service.
+
+* Souces: [bs_mirrorfull](bs_mirrorfull)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: Used by other tools like `pkglistgen` or `repocheck`
+
+#### biarchtool
+
+Manages biarch packages
+
+* Source: [biarchtool.py](biarchtool.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### build-fail-reminder
+
+Sends e-mails about packages failing to build for a long time.
+
+* Sources: [build-fail-reminder.py](build-fail-reminder.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### checknewer
+
+Checks if all packages in a repository are newer than all other repositories.
+
+* Sources: [checknewer.py](checknewer.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### deptool
+
+Assists in debugging dependencies
+
+* Sources: [deptool.py](deptool.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### requestfinder
+
+Allows to retrieve requests from OBS with quite elaborated queries.
+
+* Sources: [requestfinder.py](requestfinder.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### update_crawler
+
+Create SRs for Leap.
```

Comment:
Does it still makes sense after close gap?

Reply:
I don't think it's been in use since origin manager was introduced. @nilxam ?

Reply:
right, update_crawler should be obsoleted by origin-manager

Reply:
OK, I have marked it as obsolete. Thanks!